### PR TITLE
Delete Button and new Login Background

### PR DESCRIPTION
**Pull request by erikgarciam**
- Add new Delete button.
- Add new servlet to handle deletion of campaigns
- Seperate table into multiple tables for more information on campaigns
Screenshot 2020-07-23 at 3 53 29 PM

### CODE REVIEW

File: DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/DeleteDSACampaign.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/delete-DSACampaign")
+public class DeleteDSACampaign extends HttpServlet{
+    
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (neelbagora) [inline DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/DeleteDSACampaign.java:36]: Maybe consider merging the delete function into the existing DSA-campaigns servlet?

Comment (avenkat-google) [reply DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/DeleteDSACampaign.java:33]: +1

File: DSA-Campaign-Uplift-Estimation/src/main/webapp/Home/script-home.js [hunk 2]
```diff
@@ -13,166 +13,234 @@
...
+
+  table2.draw(data2, {showRowNumber: false, width: '100%', height: '100%'});
+
+  createThirdRow(DSACampaign, chartNumber);
+
+  console.log('Drew table.');
 }
 
-function drawDSACampaignTable(DSACampaign, chartNumber) {
+function createThirdRow(DSACampaign, chartNumber) {
```
Comment (avenkat-google) [inline DSA-Campaign-Uplift-Estimation/src/main/webapp/Home/script-home.js:215]: Change the function name? I don't quite understand the name.
And can I see a screenshot of how the page should look.

Comment (erikgarciam) [reply DSA-Campaign-Uplift-Estimation/src/main/webapp/Home/script-home.js]: Done.

File: DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/DeleteDSACampaign.java (hunk 1 above)
Comment (erikgarciam) [reply DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/DeleteDSACampaign.java]: Done.

**Pull request merged**